src/test/java/bio/terra/app/configuration/WebConfigTest.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.app.configuration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.common.category.Unit;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+@Tag(Unit.TAG)
+class WebConfigTest {
+
+  @Test
+  void swaggerUiVersionMatchesClasspath() throws IOException {
+    PathMatchingResourcePatternResolver resolver = new PathMatchingResourcePatternResolver();
+    Resource[] resources =
+        resolver.getResources("classpath:/META-INF/resources/webjars/swagger-ui-dist/*/");
+
+    assertEquals(
+        1, resources.length, "Expected one swagger-ui-dist resource, found " + resources.length);
+
+    Pattern versionPattern = Pattern.compile(".+/swagger-ui-dist/(\\d+\\.\\d+\\.\\d+)/");
+    Matcher matcher = versionPattern.matcher(((ClassPathResource) resources[0]).getPath());
+    Optional<String> currentVersion =
+        matcher.matches() ? Optional.of(matcher.group(1)) : Optional.empty();
```

Comment:
Since `matcher.results()` is already a `Stream` it provides a direct way to get an `Optional` result
```suggestion
        matcher.results().findFirst().map(m -> m.group(1));
```

You can combine it with the assert too, which avoid the separate `isPresent()` / `get()` usage, which is error prone:
```
    String currentVersion =
        matcher.results().findFirst()
            .map(m -> m.group(1))
            .orElseThrow(() -> new AssertionError("Could not find swagger-ui-dist version in classpath. Check your build.gradle dependencies."));
```